linodecli/helpers.py

```diff
@@ -40,29 +39,6 @@ def handle_url_overrides(
     ).geturl()
 
 
-def filter_markdown_links(text):
```

Comment:
This is superseded by `_markdown_links_to_rich` and apparently wasn't being consumed anywhere anyway 🤔